packages/core/src/contracts/tcr/parameterizer.ts

```diff
@@ -274,18 +248,11 @@ export class Parameterizer extends BaseWrapper<ParameterizerContract> {
    * @param fromBlock Starting block in history for events. Set to "latest" for only new events
    * @returns ParamProps for proposals that have been challenged and resolved
    */
-  public paramPropsForResolvedChallenged(fromBlock: number | "latest" = 0): Observable<ParamProp> {
+  public paramPropsForResolvedChallenged(fromBlock: number | "latest" = 0): Observable<string> {
     return this.instance
-      .NewChallengeStream({}, { fromBlock })
-      .concatFilter(async e => this.isChallengeResolved(e.args.pollID))
-      .map(e => {
-        return {
-          propID: e.args.propID,
-          paramName: e.args.name,
-          proposedValue: e.args.value,
-          pollID: e.args.pollID,
-        };
-      });
+      ._NewChallengeStream({}, { fromBlock })
+      .concatFilter(async e => this.isChallengeResolved(e.args.challengeID))
+      .map(e => e.args.propID);
```

Comment:
whats the thought around changing to just the string?

Reply:
when we updated the event to match AdChain standard, we lost a couple of the parameters in the event that help us make the ParamProp, so we just return the ID and get the ParamProp when you need it.